### ISSUE TITLE
Readability fails to keep body of content from major article

### ISSUE THREAD

**Issue by trobriander** [mozilla/readability#603]
In the following article, it gets the list of signatories (which is a table) but not the actual statement they are signing (which are a series of p elements)

https://harpers.org/a-letter-on-justice-and-open-debate/

**Comment by gijsk** [comment]
Yeah, the problem is that we've had repeated issues with ditching tables (which can be quite important for the meanings of articles - as is arguably the case here), so we can't just ignore them. And the table with signatories, ironically, contains more text than the actual statement (both more words, when splitting on whitespace, and more characters). I also suspect that the way readability scores ancestors and combines scores from descendant nodes will not have done the main text any favours here.

**Comment by trobriander** [comment]
That's exactly what's happening. The `<tr>` gets a higher score than the `<article>` tag. In fact the `<article>` is the second node in the **topCandidates** array (sorted by score) . Maybe this is being a little too primitive, but a first solution might be to see whether the next element in topCandidates is an <article> and if so, check to see whether it's an ancestor. If it passes the test, it should be the new topCandidate.

**Comment by gijsk** [comment]
Perhaps just giving article nodes a scoring bonus if there's only 1 such tag in the document... What are the respective scores, if you've just debugged this?

**Comment by trobriander** [comment]
Sure! 

`<tr>` has a score of 136
`<article>` has a score of 99
